## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 504701`
Commit: `48f2f03c6949d07b96d9f55f539579a00566199e`

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 2 packages marked as broken and skipped:</summary>
  <ul>
    <li>rstudio</li>
    <li>rstudioWrapper</li>
  </ul>
</details>
<details>
  <summary>:x: 9 packages failed to build:</summary>
  <ul>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>jellyfin-mpv-shim</li>
    <li>jellyfin-mpv-shim.dist</li>
    <li>python313Packages.mpv</li>
    <li>python313Packages.mpv.dist</li>
    <li>python314Packages.mpv</li>
    <li>python314Packages.mpv.dist</li>
    <li>switchfin</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 88 packages built:</summary>
  <ul>
    <li>ani-cli</li>
    <li>auto-editor</li>
    <li>botamusique</li>
    <li>catt</li>
    <li>catt.dist</li>
    <li>cliamp</li>
    <li>cplay-ng</li>
    <li>cplay-ng.dist</li>
    <li>curseradio</li>
    <li>curseradio.dist</li>
    <li>deno</li>
    <li>dmlive</li>
    <li>dra-cla</li>
    <li>era</li>
    <li>ff2mpv</li>
    <li>ff2mpv-go</li>
    <li>gallery-dl</li>
    <li>gallery-dl.dist</li>
    <li>gleam</li>
    <li>gonic</li>
    <li>gophertube</li>
    <li>gpodder</li>
    <li>gtk-pipe-viewer</li>
    <li>gtk-pipe-viewer.devdoc</li>
    <li>invidtui</li>
    <li>jellyfin-tui</li>
    <li>jellytui</li>
    <li>kodiPackages.sendtokodi</li>
    <li>mkchromecast</li>
    <li>mkchromecast.dist</li>
    <li>moodle-dl</li>
    <li>moodle-dl.dist</li>
    <li>mopidy-youtube (mopidyPackages.mopidy-youtube)</li>
    <li>mopidy-youtube.dist (mopidyPackages.mopidy-youtube.dist)</li>
    <li>mov-cli</li>
    <li>mov-cli.dist</li>
    <li>mpv</li>
    <li>mpvScripts.mpv-playlistmanager</li>
    <li>parrot</li>
    <li>persepolis</li>
    <li>pinchflat</li>
    <li>pipe-viewer</li>
    <li>pipe-viewer.devdoc</li>
    <li>previewqt</li>
    <li>python313Packages.bgutil-ytdlp-pot-provider</li>
    <li>python313Packages.bgutil-ytdlp-pot-provider.dist</li>
    <li>python313Packages.tubeup</li>
    <li>python313Packages.tubeup.dist</li>
    <li>python313Packages.yark</li>
    <li>python313Packages.yark.dist</li>
    <li>yt-dlp (python313Packages.yt-dlp)</li>
    <li>yt-dlp.dist (python313Packages.yt-dlp.dist)</li>
    <li>python314Packages.bgutil-ytdlp-pot-provider</li>
    <li>python314Packages.bgutil-ytdlp-pot-provider.dist</li>
    <li>python314Packages.tubeup</li>
    <li>python314Packages.tubeup.dist</li>
    <li>python314Packages.yark</li>
    <li>python314Packages.yark.dist</li>
    <li>python314Packages.yt-dlp</li>
    <li>python314Packages.yt-dlp.dist</li>
    <li>quarto</li>
    <li>quartoMinimal</li>
    <li>radio-cli</li>
    <li>scdl</li>
    <li>scdl.dist</li>
    <li>shira</li>
    <li>shira.dist</li>
    <li>somafm-cli</li>
    <li>spotdl</li>
    <li>spotdl.dist</li>
    <li>subtitleedit</li>
    <li>szurubooru.server</li>
    <li>szurubooru.server.dist</li>
    <li>tomato-c</li>
    <li>video-downloader</li>
    <li>wtwitch</li>
    <li>yewtube</li>
    <li>yewtube.dist</li>
    <li>youtube-tui</li>
    <li>ytcc</li>
    <li>ytcc.dist</li>
    <li>ytdl-sub</li>
    <li>ytdl-sub.dist</li>
    <li>ytdownloader</li>
    <li>ytfzf</li>
    <li>ytmdl</li>
    <li>ytmdl.dist</li>
    <li>ytui-music</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python313Packages.mpv</summary>
<pre>FAILED tests/test_mpv.py::TestStreams::test_play_context - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestStreams::test_python_stream - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestStreams::test_python_stream_exception - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestStreams::test_stream_open_exception - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestStreams::test_stream_open_forward - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_log_handler - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_event - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_event_shutdown - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_property_event_overflow - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_property_negative - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_property_positive - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_property_shutdown - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_shutdown - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::CommandTests::test_async_command - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::CommandTests::test_loadfile_with_subtitles - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::CommandTests::test_sub_add - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::RegressionTests::test_instance_method_property_observer - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::RegressionTests::test_unobserve_property_runtime_error - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::RegressionTests::test_wait_for_property_concurrency - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
======================== 43 failed, 4 passed in 13.14s =========================</pre>
</details>
<details>
<summary>python314Packages.mpv</summary>
<pre>FAILED tests/test_mpv.py::TestStreams::test_play_context - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestStreams::test_python_stream - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestStreams::test_python_stream_exception - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestStreams::test_stream_open_exception - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestStreams::test_stream_open_forward - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_log_handler - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_event - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_event_shutdown - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_property_event_overflow - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_property_negative - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_property_positive - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_property_shutdown - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::TestLifecycle::test_wait_for_shutdown - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::CommandTests::test_async_command - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::CommandTests::test_loadfile_with_subtitles - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::CommandTests::test_sub_add - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::RegressionTests::test_instance_method_property_observer - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::RegressionTests::test_unobserve_property_runtime_error - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
FAILED tests/test_mpv.py::RegressionTests::test_wait_for_property_concurrency - FileNotFoundError: [Errno 2] No such file or directory: &#x27;Xvfb&#x27;
======================== 43 failed, 4 passed in 16.95s =========================</pre>
</details>
<details>
<summary>switchfin</summary>
<pre>[ 93%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/presenter.cpp.o
[ 94%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/recycling_grid.cpp.o
[ 94%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/recyling_video.cpp.o
[ 95%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/search_list.cpp.o
[ 96%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/selector_cell.cpp.o
[ 96%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/svg_image.cpp.o
[ 97%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/text_box.cpp.o
[ 97%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/video_card.cpp.o
[ 98%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/video_profile.cpp.o
[ 98%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/video_progress_slider.cpp.o
1 warning generated.
[ 99%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/video_source.cpp.o
[ 99%] Building CXX object CMakeFiles/Switchfin.dir/app/src/view/video_view.cpp.o
Copying OS X content Switchfin.app/Contents/Resources/app.icns
[100%] Linking CXX executable Switchfin.app/Contents/MacOS/Switchfin
ld: library not found for -lstdc++fs
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [CMakeFiles/Switchfin.dir/build.make:1187: Switchfin.app/Contents/MacOS/Switchfin] Error 1
make[1]: *** [CMakeFiles/Makefile2:238: CMakeFiles/Switchfin.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
</details>

--- a/pkgs/by-name/de/deno/fetchers.nix
+++ b/pkgs/by-name/de/deno/fetchers.nix
@@ -11,7 +11,7 @@
     args:
     fetchurl {
       name = "librusty_v8-${args.version}";
-      url = "https://github.com/denoland/rusty_v8/releases/download/v${args.version}/librusty_v8_release_${stdenv.hostPlatform.rust.rustcTarget}.a.gz";
+      url = "https://github.com/denoland/rusty_v8/releases/download/v${args.version}/librusty_v8_simdutf_release_${stdenv.hostPlatform.rust.rustcTarget}.a.gz";
       sha256 = args.shas.${stdenv.hostPlatform.system};
       meta = {
         inherit (args) version;

--- a/pkgs/by-name/de/deno/librusty_v8.nix
+++ b/pkgs/by-name/de/deno/librusty_v8.nix
@@ -2,11 +2,11 @@
 { fetchLibrustyV8 }:
 
 fetchLibrustyV8 {
-  version = "146.1.0";
+  version = "147.0.0";
   shas = {
-    x86_64-linux = "sha256-PYCBh8+RY1nvPOKXMCns5mDRo2j0SB3Edw/ut7npjxo=";
-    aarch64-linux = "sha256-EGlTttOowHhoFBy8FQeokCnbPLi4tfkIhSek28TfcGQ=";
-    x86_64-darwin = "sha256-61d4tQ/PcNPUvDuQsMNTNUO43zLZSVEHOjdGFG0u4W8=";
-    aarch64-darwin = "sha256-BMKybPdxP9+7QD/yfbnPnFxD8N7kHPUkMcEEf4P4iSE=";
+    x86_64-linux = "sha256-PXLRowkOBRVWeonQDTN6e4BQlSLK/kobCX7eE0Y1NLY=";
+    aarch64-linux = "sha256-6jaFVmmYYOZNZmk6UHUAGyibSBQn51Ie57hB7CIZBFI=";
+    x86_64-darwin = "sha256-yTrxJ5ABXg1r0myZqy4J3m/ivYdXnsgG089PrD5cMhY=";
+    aarch64-darwin = "sha256-HVr7hNzZk4qQTct2Px7DvSJAnyq1duJCqJawY475SR4=";
   };
 }

--- a/pkgs/by-name/de/deno/package.nix
+++ b/pkgs/by-name/de/deno/package.nix
@@ -30,17 +30,17 @@ let
 in
 rustPlatform.buildRustPackage (finalAttrs: {
   pname = "deno";
-  version = "2.7.4";
+  version = "2.7.9";
 
   src = fetchFromGitHub {
     owner = "denoland";
     repo = "deno";
     tag = "v${finalAttrs.version}";
     fetchSubmodules = true; # required for tests
-    hash = "sha256-L1dLMcA7YDNMF8uRImHZlQ0mXox3kpGln6MXeB9cMLU=";
+    hash = "sha256-asRSIDpVN8sZgck5cocqfjcFNnP3CekR0lwBi0jr6GM=";
   };
 
-  cargoHash = "sha256-ebozsCns5J95u+C4smysslJB9xVltHeVOdfD2IBuLtI=";
+  cargoHash = "sha256-lL9ZeMUi5cwrqikg+GiR5hQNgWPKlpAN7yQIXSsr93k=";
 
   patches = [
     ./patches/0002-tests-replace-hardcoded-paths.patch

--- a/pkgs/by-name/de/deno/update/librusty_v8.ts
+++ b/pkgs/by-name/de/deno/update/librusty_v8.ts
@@ -26,7 +26,7 @@ const fetchArchShaTasks = (version: string, arches: Architecture[]) =>
     async (arch: Architecture): Promise<PrefetchResult> => {
       log("Fetching:", arch.nix);
       const sha256 = await run("nix-prefetch-url", [
-        `https://github.com/denoland/rusty_v8/releases/download/v${version}/librusty_v8_release_${arch.rust}.a.gz`
+        `https://github.com/denoland/rusty_v8/releases/download/v${version}/librusty_v8_simdutf_release_${arch.rust}.a.gz`
       ]);
       const sha256_sri = await run("nix-hash", ["--type", "sha256", "--to-sri", sha256]);
       log("Done:    ", arch.nix);

